packages/net/writebuffer.pony

```diff
@@ -0,0 +1,155 @@
+use "collections"
+
+class WriteBuffer
+  """
+  * A List[ByteSeq] maintained as WriteBuffer.
```

Comment:
The doc string would definitely be improved by adding some information about what the class does(provides a writeable byte buffer with convenience methods for writing byte representations of numeric types and retrieving the written bytes), as opposed to what the member variables do.

In my opinion, the example program unnecessarily obscures the intended use of class by trying to do too much. Using a random number generator to write bytes doesn't really show anything important about the class. I might almost go so far as to argue that the example program doesn't need to write to the network since that adds a lot of code that doesn't actually show how to use the class (though it does start to give some context for when one might use the class).

To specifically address two of @SeanTAllen 's questions:
- `MT()` is a random number generator from the `random` package
- `WBUDPNotify` starts writing to a socket "when it starts listening" because `listening()` is called when the socket is bound to an address, and that's when you'd start writing to the socket if the purpose of the program was to write random bytes to a socket as soon as it starts up, but maybe the real question is "Why are you writing to a socket at all?" which gets back to my point that the example may be obscuring the intention of the class

@jersey99, take a look at the doc string for the `signals` package (https://github.com/ponylang/ponyc/blob/master/packages/signals/signal.pony) for an example of a good package-level doc string.